### PR TITLE
Onboard SBSA based ARM platform to MU

### PR DESCRIPTION
**Pull request by kuqin12**
This change brings in the support for ARM based SBSA machine to QEMU virtual platform. The system supports TFA, Standalone MM and non-secure UEFI execution.

The VGA graphics are also supported, and the system is bootable to UEFI shell and Windows ARM build.

### CODE REVIEW

**Review by os-d — APPROVED** [review]

File: Platforms/QemuSbsaPkg/Docs/Features/feature_frontpage.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Front Page
+
+Todo
```
Comment (cfernald) [inline Platforms/QemuSbsaPkg/Docs/Features/feature_frontpage.md:3]: Todo

File: Platforms/QemuSbsaPkg/PlatformBuild.py [hunk 2]
```diff
@@ -0,0 +1,575 @@
+# @file
+# Script to Build ArmVirtPkg UEFI firmware
```
Comment (kuqin12) [inline Platforms/QemuSbsaPkg/PlatformBuild.py:2]: Not really true

File: Platforms/QemuSbsaPkg/Library/MsPlatformDevicesLibQemuSbsa/MsPlatformDevicesLib.inf [hunk 3]
```diff
@@ -0,0 +1,45 @@
+## @file
+# QemuQ35 Ms Platform Devices Library
+#
+# Copyright (C) Microsoft Corporation. All rights reserved.
```
Comment (kuqin12) [inline Platforms/QemuSbsaPkg/Library/MsPlatformDevicesLibQemuSbsa/MsPlatformDevicesLib.inf:4]: consistency

File: Platforms/QemuSbsaPkg/Library/MsPlatformDevicesLibQemuSbsa/MsPlatformDevicesLib.c [hunk 4]
```diff
@@ -0,0 +1,1280 @@
+/** @file
+ *MsPlatformDevicesLib  - Device specific library.
+
+Copyright (C) Microsoft Corporation. All rights reserved.
```
Comment (kuqin12) [inline Platforms/QemuSbsaPkg/Library/MsPlatformDevicesLibQemuSbsa/MsPlatformDevicesLib.c:4]: Consistency

**Comment by cfernald** [comment]
Kun and I discussed offline. Eventually we think we will make a second change to pull common QEMU code, tools, and documentation into a new QemuPkg that could be consumed by both Q35 and SBSA (or potentially others in the future), but since this pkg will be under frequent changes for the time being it might be best to avoid refactoring Q35 based on this package until we are more stable.

File: Platforms/QemuSbsaPkg/Docs/Features/feature_frontpage.md (hunk 1 above)
Comment (kuqin12) [reply Platforms/QemuSbsaPkg/Docs/Features/feature_frontpage.md:3]: Discussed offline, will resolve them when merging the documents into the common package.

**Review by cfernald — APPROVED** [review]

File: Platforms/QemuSbsaPkg/PlatformBuild.py (hunk 2 above)
Comment (kuqin12) [reply Platforms/QemuSbsaPkg/PlatformBuild.py:2]: Updated.

File: Platforms/QemuSbsaPkg/Library/MsPlatformDevicesLibQemuSbsa/MsPlatformDevicesLib.c (hunk 4 above)
Comment (kuqin12) [reply Platforms/QemuSbsaPkg/Library/MsPlatformDevicesLibQemuSbsa/MsPlatformDevicesLib.c:4]: Updated.

File: Platforms/QemuSbsaPkg/Library/MsPlatformDevicesLibQemuSbsa/MsPlatformDevicesLib.inf (hunk 3 above)
Comment (kuqin12) [reply Platforms/QemuSbsaPkg/Library/MsPlatformDevicesLibQemuSbsa/MsPlatformDevicesLib.inf:4]: Updated.

**Pull request merged**